uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.controller.js

```diff
@@ -468,6 +478,31 @@ define([
     }
 
     /**
+     * Validates the uploaded file size and
+     * displays error mesage for large files
+     *
+     * @param  {object} uploader
+     * @return {boolean}
+     */
+    function validateFileSize (uploadedFiles) {
```

Comment:
Name of the function is `validateFileSize`, but the param is `uploadedFile**s**` (singular and plural, inconsistent)

Reply:
The function accept params as `[list of files]`. I think we should use `files` as param?

Reply:
Yep, "files" work. Also function should be "validateFilesSizes"